testnet/stacks-node/Cargo.toml

```diff
@@ -31,6 +31,7 @@ wsts = { workspace = true }
 rand = { workspace = true }
 rand_core = { workspace = true }
 hashbrown = "0.14"
+bincode = "1.3.3"
```

Comment:
Why is this now a depedency?

Reply:
I'd very strongly prefer that `bincode` was kept only within the signer.  If you want the node proper to consume signer messages, then they need to be structs that implement `StacksMessageCodec`.  In particular, `bincode` does not by default guard against memory exhaustion errors that result from deserializing malicious inputs (whereas `StacksMessageCodec` does).

Reply:
This is a depenency because this is how the messages are serialized in libsigner. if we do not deserialize it the same way, then it will fail. I can add StacksMessageCodec to all the types but that adds a lot of what I would consider unnecessary code. However if the desire to remove the bincode dependency outweighs the con of a ton of extra code, can do.

Reply:
Done! :D

Reply:
Done!